routes.yaml

```diff
@@ -0,0 +1,9 @@
+group: streampath
```

Comment:
Please remove this file; we will be deploying this application to ECS as a container.